<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSourceSelectorDialogForm</class>
 <widget class="QDialog" name="DataSourceSelectorDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Source Selector</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>289</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="te::qt::widgets::HelpPushButton" name="m_helpPushButton">
        <property name="text">
         <string>&amp;Help</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="m_selectPushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Select</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_cancelPushButton">
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_selectPushButton</tabstop>
  <tabstop>m_cancelPushButton</tabstop>
  <tabstop>m_helpPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_cancelPushButton</sender>
   <signal>pressed()</signal>
   <receiver>DataSourceSelectorDialogForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <CustomWidgets>
	<CustomWidget>
	 <class>te::qt::widgets::HelpPushButton</class> 
	 <extends>QPushButton</extends> 
	 <header>terralib/qt/widgets/help/HelpPushButton.h</header>
	</CustomWidget>
 </CustomWidgets>
</ui>
